handbook/Best practices/React/Organization.md

```diff
@@ -4,11 +4,62 @@
 
 This section describes a sensible starting point for organizing a React application that supports server-side rendering.
 
-## `/app`
+## Application
 
-### `/app/components` and `<Component>/components`
```

Comment:
Can we still leave these top-level folder paths in? I think there useful to have.